garfieldbot/garfieldbot.py

```diff
@@ -0,0 +1,45 @@
+# bot.py
+import os
+import discord
+import requests
+import logging
+import json
+import random
+
+#Logging configuration
+logging.basicConfig(format='%(levelname)s %(asctime)s - %(message)s', level=logging.INFO)
+
+TOKEN = os.getenv('DISCORD_TOKEN')
+TENOR_TOKEN = os.getenv('TENOR_TOKEN')
+GUILD = 'What Are The Odds?!'
+channel_id = 402916062452252675
+
+intents = discord.Intents.default()
+intents.members = True
+client = discord.Client(intents=intents)
+
+def tenorgif():
+    search_term = "garfield"
```

Comment:
Perhaps an enhancement would be to make the search term Garfield + the day of the week? That way we're not getting the same 20 Garfield gifs to choose from.

Reply:
> Perhaps an enhancement would be to make the search term Garfield + the day of the week? That way we're not getting the same 20 Garfield gifs to choose from.

Yes was planning on adding this next - good point

Reply:
We'll have to groom this for sure. Say 5 points?